ax/modelbridge/torch.py

```diff
@@ -120,7 +120,7 @@ def __init__(
                 `model_gen_options` passed to the `Adapter.gen` method.
             torch_device: The device to use for any torch tensors and operations
                 on these tensors.
-            data_loader_options: A dictionary of options for loading data.
+            data_loader_config: A dictionary of options for loading data.
```

Comment:
This also isn't a dictionary, it has to be an object which is what I was interested in per second comment of https://github.com/facebook/Ax/issues/3659